draft-ietf-quic-manageability.md

```diff
@@ -284,12 +284,24 @@ future versions of the protocol.
 
 ## Connection confirmation {#sec-confirm}
 
-Connection establishment requires cleartext packets and is using a TLS handshake
-on stream 0. Therefore it is detectable using heuristics similar to those used
-to detect TLS over TCP. 0-RTT connection may additional also send data packets,
-right after the Client Initial with the TLS client hello. These data may be
-reordered in the network, therefore it may be possible that 0-RTT Protected data
-packets are seen before the Client Initial packet.
+Connection establishment uses Client Initial, Server Cleartext, and Client
```

Comment:
We just changed the name of these packets.  Sorry.

Reply:
NP, was tracking -07. 

The change also changes how handshake tracking works, which I'll address in a separate PR.